static/css/components/buttonSearch.less

```diff
@@ -0,0 +1,51 @@
+/**
+ * Search button and text input
+ * [TODO: Link to documentation]
+ */
+
+.search-div {
+  // @grey-fafafa;
+  background-color: @white;
+  border: 1px solid @dark-beige;
+  border-radius: .3em;
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: .2%;
+  padding-left: .5%;
+  width: 50%;
+}
+
+.search-btn-icon {
+  background: url(/static/images/search-icon.svg) center no-repeat;
+  width: 35px;
+  height: 45px;
+  cursor: pointer;
+  border: 0;
+  flex: 0 0 auto;
+}
+
+.search-input {
+  margin: 0 !important;
+  border: 0;
+  outline: none;
```

Comment:
Yeah let's drop these outline overrides ; we want the native browser ones for accessibility.